setup.php

```diff
@@ -250,7 +250,7 @@ function cloneClicked() {
 
         case 2:
             echo "<b>Step $state</b><br><br>\n";
-            echo "Now you need to supply the MySQL server information and path information. Detailed instructions on each item can be found in the <a href='Documentation/INSTALL' target='_blank'><span STYLE='text-decoration: underline;'>'INSTALL'</span></a> manual file.
+            echo "Now you need to supply the MySQL server information and path information. Detailed instructions on each item can be found in the <a href='Documentation/INSTALL' rel="noopener" target='_blank'><span STYLE='text-decoration: underline;'>'INSTALL'</span></a> manual file.
```

Comment:
Also, travis caught a parsing error here:
https://travis-ci.org/openemr/openemr/jobs/473458071
(need to use single quotes here rather than the double quotes)

Reply:
Actually, you'll need to also do this with all the other changes in this setup.php script :)

Reply:
Okay, I think I've handled this one!